bootstrap.js

```diff
@@ -12,7 +12,12 @@ const { classes: Cc, interfaces: Ci, results: Cr, utils: Cu } = Components;
 
 var Allow = (function () {
 	function wildcard (string) {
-		return new RegExp("^" + string.replace(/\./g, "\\.").replace(/\*/g, ".*?").replace(/\?/g, ".") + "$");
+		// Escape special characters that are special in regular expressions
+		string = string.replace(/(\\|\^|\$|\{|\}|\(|\)|\+|\||\<|\>|\&|\.)/g, "\\$1");
```

Comment:
No, that makes sense, just didn't know that was your intention, could you add a comment about that so that nobody ends up adding it and breaking everything?